model_clone/mixins/clone.py

```diff
@@ -320,8 +320,12 @@ def make_clone(self, attrs=None, sub_clone=False):
 
         # Clone one to many/many to one fields
         for field in many_to_one_or_one_to_many_fields:
-            getattr(duplicate, field.related_name).set(
-                getattr(self, field.related_name).all())
+            for item in getattr(self, field.related_name).all():
```

Comment:
Could this use `.set` instead.

```
items = []
for ...:
   items.append(...)

getattr(duplicate, field.related_name).set(items)
```

Reply:
Yes it could. Then it is required to create an intermediate list to store the cloned related objects and then set it to the set.

Reply:
Should I change it to it?

Reply:
Yeah that's fine.